src/translations/en.global.json

```diff
@@ -8,5 +8,19 @@
     "account.create.success": "Congrats! This name is available.",
     "account.create.error": "Username is taken. Try something else.",
     "account.available.success": "User found.",
-    "account.available.error": "User not found."
+    "account.available.error": "User not found.",
+
+    "actions": {
+       "CreateAccount": "New account created: ${receiverId}",
+       "DeleteAccount": "Account deleted: ${receiverId}",
+       "DeployContract": "Contract deployed: ${receiverId}",
+       "FunctionCall": "Called method in contract: ${methodName}",
```

Comment:
Let's change this to ```Called method in contract: ${receiverId}``` to keep consistent with Explorer (and since we don't have access to method names yet)

Reply:
we have access to method names, so we should display both.

Reply:
We currently do? If that's the case, we can display ```Called method ${methodName} in contract @${receiverId}```

The current implementation doesn't seem to have method access: [Screenshot](https://drive.google.com/file/d/1EWc2DfPpiAGkg9GF84n43gBwmuIHE4r2/view?usp=sharing)

Reply:
> The current implementation doesn't seem to have method access: Screenshot

maybe it's lost somewhere in the plumbing, but it's totally is available in transaction itself and so needs to be displayed (both in wallet and in explorer)